UPGRADE.md

```diff
@@ -1,5 +1,42 @@
 # Upgrade to 2.10
 
+## Deprecated `Doctrine\DBAL\Event\ConnectionEventArgs` methods
```

Comment:
A lot of the deprecations mentioned in `UPGRADE.md` aren't in the diff: are these already deprecated? (Can't check from the phone)

Reply:
Yeah, I forgot to add some of them to phpDoc. Fixed.